README.md

```diff
@@ -249,6 +249,53 @@ the autocomplete/ctr file in your `.bashrc`, or manually like:
 $ source ./contrib/autocomplete/ctr
 ```
 
+### CRI
+
+`cri` is a [containerd](https://containerd.io/) plugin implementation of Kubernetes [container runtime interface (CRI)](https://github.com/kubernetes/cri-api/blob/master/pkg/apis/runtime/v1alpha2/api.proto).
+
+With it, you could run Kubernetes using containerd as the container runtime.
+![cri](./docs/cri.png)
+
+#### Current Status
+
+`cri` is a native plugin of containerd 1.1 and above. It is built into containerd and enabled by default.
+
+>  **Note:** `cri` as of containerd 1.5, the cri plugin is merged into the containerd/containerd repo. For example, the source code previously stored under [`containerd/cri/pkg`](https://github.com/containerd/cri/tree/release/1.4/pkg)
+was moved to [`containerd/containerd/pkg/cri` package](https://github.com/containerd/containerd/tree/master/pkg/cri).
+
+`cri` is in GA:
```

Comment:
"The `cri` plugin has reached GA status, representing that it is:"

then you can drop the "It [is]" from the bullets and instead have:
 * Feature complete
 * Works with Kubernetes 1.10 and above 
 * Passes all ..